src/main/java/medicaldoctor/core/AppSession.java

```diff
@@ -21,14 +22,15 @@
     public static Encryption ENCRYPTION = new Encryption();
 
     private static User activeUser;
-
+    public static ControllerManager cm;
```

Comment:
```suggestion
    public static ControllerManager CONTROLLER_MANAGER;
```

also can move it up with the other public vars :)